sys/net/application_layer/nanocoap/Makefile.include

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter nanocoap_dtls,$(USEMODULE)))
+  CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT := $(or $(CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT),2)
+  DTLS_MAX_BUF ?= ((1 << ($(CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT) + 3)) + 36)
```

Comment:
You'll need to escape the whitespace and special characters, otherwise the shell will try to interpret them

Reply:
I  tested this and riotbuild.h was generated correctly...

Reply:
Ah you are right, must have messed something up when I tried it